unstructured/partition/common.py

```diff
@@ -183,6 +186,69 @@ def normalize_layout_element(
         )
 
 
+def set_hierarchy_by_indentation(  # Add exception handling for elements without coordinates
+    elements: List[Element],
+) -> List[Element]:
+    left = []
+    right = []
+    coordinates = []
+    for ele in elements:
+        coordinates.append(getattr(ele.metadata.coordinates, "points")[0])
+
+    coords_np = np.asarray(coordinates)
+    median = np.median(coords_np)
+
+    # Split text from multicolumn page in left and right lists
+    # Will not work for more than 2 columns
+    for ele in elements:
+        if ele.metadata.coordinates.points[0][0] <= median:
+            left.append((ele, "left"))
+        else:
+            right.append((ele, "right"))
+
+    # Assign parent id to indented text
+    prev_ele = []
```

Comment:
This looks like a way of assigning a null type to `prev_ele`, and then `prev_ele` gets updated after the first run through the code. I think `None` would be a better null type to assign rather than an empty list.